sdk/java-sdk/src/main/scala/com/akkaserverless/javasdk/impl/action/ActionsImpl.scala

```diff
@@ -123,10 +146,6 @@ final class ActionsImpl(_system: ActorSystem, services: Map[String, ActionServic
         throw new RuntimeException(s"Unknown metadata implementation: ${other.getClass}, cannot send")
     }
 
-  private def toProtocol(ex: Throwable): ActionResponse = {
-    ActionResponse(ActionResponse.Response.Failure(Failure(0, ex.getMessage)))
```

Comment:
ok, so this `ex.getMessage` was the leak (via proxy) out to the client?

Reply:
Yeah, but there are more places for the other component types, I just started out with actions as a first step